_posts/2019-04-16-accessing-kafka-part-1.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title:  "Accessing Kafka: Part 1 - Introduction"
+date: 2019-04-16
+author: jakub_scholz
+---
+
+Scalability is one of the flagship features of Apache Kafka.
+It is achieved by partitioning the data and distributing them across multiple brokers.
+The data sharding has also a big impact on how Kafka clients connect to the brokers.
+This is especially visible when running Kafka inside a platform like Kubernetes but is accessed from outside of the platform.
+This blog-series will explain how Kafka and its clients work and how Strimzi deals with it to make it accessible for clients running outside of Kubernetes.
+
+<!--more-->
+
+It would be of course not sufficient to just shard the data into partitions.
+Also the ingress and egress data traffic needs to be properly sharded.
+The clients writing or reading from given partition have to connect directly to the broker which is hosting it.
+Thanks to the clients connecting directly to the individual brokers, the brokers don't need to do any forwarding of data between the clients and the partition it self.
```

Comment:
The partition is not really a participant in any network exchange, so this doesn't really make sense.

```suggestion
Thanks to the clients connecting directly to the individual brokers, the brokers don't need to do any forwarding of data between the clients and other brokers.
```